lib/routes/sign.js

```diff
@@ -101,10 +103,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
         ).then(function(certResult) {
           log.activityEvent('account.signed', request, {
             uid: uid,
+            network: getNetworkHash(uid, request.info.remoteAddress),
+            device: getDeviceHash(sessionToken.id),
             account_created_at: sessionToken.accountCreatedAt
           })
```

Comment:
We may be better served by a separate event, rather than adding extra data to `account.signed`.  The `account.signed` events are persisted, exported to files, analyzed for dashboards, etc.  Using a distinct event means we can have more control over retention and processing policies.